### PR TITLE
Fix labeler workflow

### PR DESCRIPTION
**Pull request by jtpio**
## References

Follow-up to https://github.com/jupyterlab/jupyterlab/pull/10306

## Code changes

None

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -9,5 +9,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/labeler@main
-      env:
+      with:
```
Comment (goanpeca) [inline .github/workflows/labeler.yml:12]: Whoops, I messed that one. Than ks @jtpio

**Review by goanpeca — APPROVED** [review]

**Comment by jtpio** [comment]
I guess this needs to be merged into master first to be taken into account?

**Review by blink1073 — APPROVED** [review]

**Comment by goanpeca** [comment]
>I guess this needs to be merged into master first to be taken into account?

Yes, because it runs with the `pull_request_target` I think.

**Comment by goanpeca** [comment]
Will go ahead an merge since it is a simple fix. Thanks @jtpio !

**Pull request merged**